cora/util/cubicspline.pyx

```diff
@@ -97,7 +99,7 @@ cdef class Interpolater(object):
         self._n = <Py_ssize_t>self.__data_.shape[0]
 
 
-    def value(self, x):
+    def value(self, int x):
```

Comment:
Why did you choose the `int` type for the inputs to these?

Reply:
More broadly, I think you can't type this at all. It needs to work for both arrays and Python scalars, so you need the flexible Python typing to allow that. That's the point of the `value_cdef` method is to be where the typing gets enforced.